entrypoint.sh

```diff
@@ -54,7 +54,7 @@ for namespace in $NAMESPACES; do
       label_selector="-l OWNER!=TILLER"
     fi
 
-    /kubectl --namespace="${namespace}" get --export -o=json "$type" "$label_selector" | jq --sort-keys \
+    /kubectl --namespace="${namespace}" get --export -o=json "$type" $label_selector | jq --sort-keys \
```

Comment:
The value may contain space (`-l OWNER!=TILLER`) and when it is quoted bash treats that as single argument (not parameter), which incorrect